lib/board.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Board
+  @@winner_points = [0, 1, 2, 3, 4, 5, 6, 7, 8, 0, 3, 6, 1, 4, 7, 2, 5, 8, 0, 4, 8, 6, 4, 2]
```

Comment:
What do you think about just making this an instance variable instead of a class variable as you did here (i.e, placing it inside the `initialize` method)? Anyway it's a data that's only useful for the instance of `Board`.